vite.config.ts

```diff
@@ -3,5 +3,8 @@ import react from '@vitejs/plugin-react'
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [react()],
+    plugins: [react()],
+    optimizeDeps: {
```

Comment:
remove this, you were getting the mapbox error because you were importing some backend code in your react code and that backend code was using bcrypt, which cannot run in the browser